index.html

```diff
@@ -0,0 +1,193 @@
+                    <li class="angagement">Engage in hands-on workshops facilitated by experts in various fields rather than watching them.</li>
+                </ul>
+            </li>
+            <li class="box box5">
+                <ul>
+                    <li class="activity ignite long">CC ignite</li>
+                    <li class="angagement angagement2">Get opportunities to network with CC affiliates around the world, also after the summit.</li>
+                </ul>
+            </li>
+        </ul>
+        <button id="join" class="join">Join CC Global Summit 2023</button>
+        <div id="join2">SEE THE WHOLE PROGRAM</div>
+        <hr class="hr0">
+        <br>
+        <br>
+        <br>
+    </section> 
+    <div class="featured">Featured speakers </div>
+    <br>
+    <hr class="hr1">
+    <ul id="speakers">
+
+    </ul> 
+    <button class="see-more">MORE</button>
+    <button class="see-less">See Less</button>
+    <div id="reserved">
+        <div class="logo3">Youth Leadership Global Summit<div class="mylogo mylogo2">YLGS</div>
+    </div>
+    <div class="reservation">2023 Youth Leadership Kenya. <br> Some Rights Reserved. <br> If you need to make any changes or have any questions, please feel free to contact us.
+    </div>
+    </div>
+    <!-- About-section -->
+    <div class="mobile-version version2">
+        <div class="hamburger"><img src="images/images/mobile_menu 1.png" alt="menu"></div>
+        <div class="links links-inactive">
+            <span class="close">&times</span>
+            <ul>
+                <li><a href="#about-section" class="nav-link">About</a></li>
+                <li><a href="#program" class="nav-link">Program</a></li>
+                <li><a href="#join" class="nav-link">Join</a></li>
+                <li><a href="#sponsor" class="nav-link">Sponsor</a></li>
+                <li><a href="#news" class="nav-link">News</a></li>
+            </ul>
+        </div>
+    </div>
+    <div class="desktop-version">
+        <ul id="first-nav">
+            <li><a href="#facebook"><img src="images/images/Rectangle.png" alt="facebook"></a></li>
+            <li><a href="#twitter"><img src="images/images/twitter (1).png" alt="twitter"></a></li>
+            <li class="eng">English</li>
+            <li class="eng">My page</li>
+            <li class="eng">Logout</li>
+        </ul>
+        <ul id="second-nav">
+             <a href="#homepage" class="desk-logo"><img src="images/images/logo.png" alt="logo"></a>
+            <li><a href="#about-section" class="nav-link">About</a></li>
+            <li><a href="#program" class="nav-link">Program</a></li>
+            <li><a href="#join" class="nav-link">Join</a></li>
+            <li><a href="#sponsors" class="nav-link">Sponsor</a></li>
+            <li><a href="#join2" class="nav-link">News</a></li>
+            <li class="navbar-link campaign">YLS Campaign</li>
+        </ul>
+    </div>
+    <section  id="about-section">
```

Comment:
please as requested before this should not be a section in the project requirements you were asked to create a new page a separate html file for the about page and the link in the header should redirect the user the about page.